### ISSUE TITLE
[Bug] ci_cpu image failing in `ubuntu_install_vitis_ai_packages_ci.sh`

### ISSUE THREAD

**Issue by leandron** [apache/tvm#9445]
During last night's Docker image rebuild, there is a new error happening in the context of `ubuntu_install_vitis_ai_packages_ci.sh`:

```
 ---> Running in 2b79c8b7223a
Collecting progressbar
  Downloading https://files.pythonhosted.org/packages/a3/a6/b8e451f6cff1c99b4747a2f7235aa904d2d49e8e1464e0b798272aa84358/progressbar-2.5.tar.gz
Requirement already satisfied: h5py==2.10.0 in /usr/local/lib/python3.6/dist-packages (2.10.0)
Requirement already satisfied: six in /usr/local/lib/python3.6/dist-packages (from h5py==2.10.0) (1.16.0)
Requirement already satisfied: numpy>=1.7 in /usr/local/lib/python3.6/dist-packages (from h5py==2.10.0) (1.19.5)
Building wheels for collected packages: progressbar
  Building wheel for progressbar (setup.py): started
  Building wheel for progressbar (setup.py): finished with status 'done'
  Created wheel for progressbar: filename=progressbar-2.5-cp36-none-any.whl size=12082 sha256=f08fd19649c515301885e362e456a27a234ab567f2e125c0c30ee03c6e094d4f
  Stored in directory: false/wheels/c0/e9/6b/ea01090205e285175842339aa3b491adeb4015206cda272ff0
Successfully built progressbar
Installing collected packages: progressbar
Successfully installed progressbar-2.5
WARNING: You are using pip version 19.3.1; however, version 21.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Cloning into '/opt/pyxir'...
Note: checking out '506e8ac176c04b5d7e75ffe7ecbb1f2c4caa6498'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

Submodule 'lib/Catch2' (https://github.com/catchorg/Catch2.git) registered for path 'lib/Catch2'
Submodule '3rdparty/cvx' (https://github.com/jtuyls/cvx.git) registered for path 'lib/cvx'
Submodule 'lib/pybind11' (https://github.com/pybind/pybind11.git) registered for path 'lib/pybind11'
Cloning into '/opt/pyxir/lib/Catch2'...
Cloning into '/opt/pyxir/lib/cvx'...
Cloning into '/opt/pyxir/lib/pybind11'...
Submodule path 'lib/Catch2': checked out 'bed47374cee07c6f9fec1a84d1afddba887fe731'
Submodule path 'lib/cvx': checked out 'ede40c895a48103312aabdecc4ffbc8f97c711c8'
Submodule path 'lib/pybind11': checked out '4f72ef846fe8453596230ac285eeaa0ce3278bb4'
Submodule 'tools/clang' (https://github.com/wjakob/clang-cindex-python3) registered for path 'lib/pybind11/tools/clang'
Cloning into '/opt/pyxir/lib/pybind11/tools/clang'...
Submodule path 'lib/pybind11/tools/clang': checked out '6a00cbc4a9b8e68b71caf7f774b3f9c753ae84d5'
running install
running bdist_egg
running egg_info
creating python/pyxir.egg-info
writing python/pyxir.egg-info/PKG-INFO
writing dependency_links to python/pyxir.egg-info/dependency_links.txt
writing requirements to python/pyxir.egg-info/requires.txt
writing top-level names to python/pyxir.egg-info/top_level.txt
writing manifest file 'python/pyxir.egg-info/SOURCES.txt'
/usr/lib/python3.6/distutils/dist.py:261: UserWarning: Unknown distribution option: 'extra_require'
  warnings.warn(msg)
/usr/local/lib/python3.6/dist-packages/setuptools/command/install.py:37: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.

 setuptools.SetuptoolsDeprecationWarning,
/usr/local/lib/python3.6/dist-packages/setuptools/command/easy_install.py:159: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.

 EasyInstallDeprecationWarning,

/usr/local/lib/python3.6/dist-packages/setuptools/dist.py:151: UserWarning: newlines not allowed and will break in the future

  warnings.warn("newlines not allowed and will break in the future")

Traceback (most recent call last):
  File "setup.py", line 318, in <module>
    zip_safe=False
  File "/usr/local/lib/python3.6/dist-packages/setuptools/__init__.py", line 159, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/install.py", line 74, in run
    self.do_egg_install()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/install.py", line 116, in do_egg_install
    self.run_command('bdist_egg')
  File "/usr/lib/python3.6/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/bdist_egg.py", line 155, in run
    self.run_command("egg_info")
  File "/usr/lib/python3.6/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/egg_info.py", line 299, in run
    self.find_sources()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/egg_info.py", line 306, in find_sources
    mm.run()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/egg_info.py", line 541, in run
    self.add_defaults()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/egg_info.py", line 578, in add_defaults
    sdist.add_defaults(self)
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/py36compat.py", line 32, in add_defaults
    self._add_defaults_python()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/sdist.py", line 113, in _add_defaults_python
    self._add_data_files(self._safe_data_files(build_py))
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/egg_info.py", line 621, in _safe_data_files
    return build_py.get_data_files_without_manifest()
  File "/usr/lib/python3.6/distutils/cmd.py", line 103, in __getattr__
    raise AttributeError(attr)
AttributeError: get_data_files_without_manifest
The command '/bin/sh -c bash /install/ubuntu_install_vitis_ai_packages_ci.sh' returned a non-zero code: 1
ERROR: docker build failed.
script returned exit code 1
```

cc @jtuyls to help troubleshoot when possible

**Comment by jtuyls** [comment]
Seems to be an issue introduced by setuptools version 58.5. See https://github.com/pypa/setuptools/issues/2849 and https://github.com/scikit-learn/scikit-learn/pull/21549. We can fix it by pinning the setuptools version for now? I am trying out setuptools v58.4.0 right now and can create a PR for that.

**Comment by leandron** [comment]
> Seems to be an issue introduced by setuptools version 58.5. See [pypa/setuptools#2849](https://github.com/pypa/setuptools/issues/2849) and [scikit-learn/scikit-learn#21549](https://github.com/scikit-learn/scikit-learn/pull/21549). We can fix it by pinning the setuptools version for now? I am trying out setuptools v58.4.0 right now and can create a PR for that.

Yes, I'm ok with pinning setuptools. Thanks for looking into it.

**Comment by leandron** [comment]
Fixed by #9446.
